store.py

```diff
@@ -178,13 +178,6 @@ def update_from_board(self):
                     "Part %s does not exist in the database and will be created from the board.",
                     board_part["reference"],
                 )
-<<<<<<< HEAD
```

Comment:
you could do a rebase and fixup this commit into the earlier one where the merge conflict was introduced, if you are comfortable with using git like that.